### ISSUE TITLE
Test all commands output

### ISSUE THREAD

**Issue by ingydotnet** [ingydotnet/git-subrepo#44]
We need to test the output of commands when they succeed, and also fail.

We should also test `--verbose` and `--debug` output.

**Comment by grimmySwe** [comment]
@ingydotnet I agree on testing normal output, testing verbose and debug seems to me a lot of work for very little benefit.
